client/idrepo/enduser/src/main/resources/org/apache/syncope/client/enduser/pages/SelfResult_ja.properties

```diff
@@ -14,4 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-login=Back to Home
+login=????
```

Comment:
Are those real question marks?

Reply:
no, I think that this is an IDE issue. Going to replace with correct chars